tests/Unit/ApparentHorizons/Test_Strahlkorper.cpp

```diff
@@ -123,6 +124,30 @@ void test_physical_center() {
   }
 }
 
+void test_radius() {
+  const double y11_amplitude = 1.0;
+  const double radius = 2.0;
+  const std::array<double, 3> center = {{0.1, 0.2, 0.3}};
+  const auto strahlkorper =
+      create_strahlkorper_y11(y11_amplitude, radius, center);
+
+  // Now construct a Y00 + Im(Y11) surface by hand.
+  const auto theta_phi = strahlkorper.ylm_spherepack().theta_phi_points();
+  const auto& theta_points = strahlkorper.ylm_spherepack().theta_points();
+  const auto& phi_points = strahlkorper.ylm_spherepack().phi_points();
```

Comment:
Is the first of these three redundant with the other two?

Reply:
Basically. It was used to get n_points, but I could just get it by multiplying size of theta_points and size of phi_points